src/Features/Core/Portable/ExternalAccess/UnitTesting/SolutionCrawler/UnitTestingPredefinedInvocationReasons.cs

```diff
@@ -6,24 +6,21 @@ namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.SolutionCrawler
 {
     internal static class UnitTestingPredefinedInvocationReasons
     {
-        public const string SolutionRemoved = nameof(SolutionRemoved);
-
-        public const string ProjectParseOptionsChanged = nameof(ProjectParseOptionsChanged);
+        public const string SemanticChanged = nameof(SemanticChanged);
+        public const string Reanalyze = nameof(Reanalyze);
         public const string ProjectConfigurationChanged = nameof(ProjectConfigurationChanged);
 
-        public const string DocumentAdded = nameof(DocumentAdded);
-        public const string DocumentRemoved = nameof(DocumentRemoved);
+        public const string HighPriority = nameof(HighPriority);
```

Comment:
I think this is also unused.

Reply:
currently it is subtle used.  see the above explanation.